src/Raven.Client/Documents/Operations/Revisions/RevisionsBinConfiguration.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using Raven.Client.Documents.Conventions;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.Revisions
+{
+    public sealed class RevisionsBinConfiguration : IFillFromBlittableJson, IDynamicJson
+    {
+        public bool Disabled { get; set; }
+        public TimeSpan? MinimumEntriesAgeToKeep { get; set; }
+        public TimeSpan RefreshFrequency { get; set; } = TimeSpan.FromMinutes(5);
+        public TimeSpan CleanupInterval { get; set; } = TimeSpan.FromSeconds(15);
+        public int MaxItemsToProcess { get; set; } = int.MaxValue;
```

Comment:
wondering if we should think about safer default? e.g. 16 * 1024

Reply:
Done